ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -308,9 +369,10 @@ func getTaskMetadataErrorResponse(endpointContainerID, requestType string, err e
 		field.Error:       err,
 		field.RequestType: requestType,
 	})
-	return http.StatusInternalServerError, fmt.Errorf("failed to get task metadata due to internal server error for container: %s", endpointContainerID)
+	return http.StatusInternalServerError, fmt.Errorf("failed to get task metadata due to internal server error for container: %s with error: %v", endpointContainerID, err)
```

Comment:
This will potentially expose internal errors to customers(?) I think we should avoid that.

Reply:
Makes sense, will omit this.